lib/libthr/thread/thr_private.h

```diff
@@ -289,7 +289,9 @@ struct pthread_attr {
 	int	flags;
 	void	*stackaddr_attr;
 	size_t	stacksize_attr;
```

Comment:
Humm, we already have padding earlier in the structure since it starts with 5 int's before having a pointer, even on LP64.  I also think it doesn't really matter for this structure.